examples/connection/typescript/returnArgumentsArray.ts

```diff
@@ -0,0 +1,62 @@
+import { FieldInfo, Query } from 'mysql';
```

Comment:
Should these be exported from `promise-mysql` rather than the user having to import them from the underlying mysql library?

Reply:
Sure why not. I 'll do it the same way the  type of ```mysqlModule``` is exported from the ```index.d.ts``` file